lib/exceptions.rb

```diff
@@ -6,4 +6,5 @@ class RBACError < StandardError; end
   class KieError < StandardError; end
   class InvalidStateTransitionError < StandardError; end
   class UserError < StandardError; end
+  class NegativeSequenceError < StandardError; end
```

Comment:
Even with retry, this error still has chance to be raised. Should add it in the mapping: https://github.com/RedHatInsights/approval-api/blob/master/config/initializers/custom_exception_mappings.rb#L2